src/ogd/games/BLOOM/features/FailCount.py

```diff
@@ -25,20 +25,23 @@ def _featureFilter(cls, mode: ExtractionMode) -> List[str]:
         return []
 
     def _updateFromEvent(self, event: Event) -> None:
-        lose_condition = event.EventData.get("lose_condition", "")
-        if lose_condition:
-            if lose_condition in self.fail_counts:
-                self.fail_counts[lose_condition] += 1
+        fail_type = event.EventData.get("lose_condition", "")
+        if fail_type in self.fail_type_counts:
+            self.fail_type_counts[fail_type] += 1
```

Comment:
This looks good logic-wise, just for safety's sake could we throw in an `else` case to print out a warning if we got an unrecognized `fail_type`?